boxes/forms.py

```diff
@@ -1,12 +1,26 @@
 from django.forms import ModelForm, Form, CharField, Textarea
 from .models import Box
+from django.utils import timezone
 
 
 class CreateBoxForm(ModelForm):
     class Meta:
         model = Box
         fields = ["name", "description", "expires_at"]
 
+    def clean_expires_at(self):
+        # Validate the expiration date
+        if self.expires_at:
```

Comment:
@dethos The code is failing with: `AttributeError: 'CreateBoxForm' object has no attribute 'expires_at'`. Should I only check the `cleaned_data`?